views/homepage.ejs

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+
+<head> 
+</head>
+
+<body>
+    
+    <main>
+        <div id="review">
+            <form action="/homepage" method="POST">
+                <h2>Reviews</h2>
+                    <h2><%= review.name%></h2>
+                    <h3><%= review.review %></h3>
+
+                <br><br>
```

Comment:
In plaats van < br > kan je hier ook een < div > gebruiken om dit te wrappen. Dan geef je die display: flex en flex-direction: column en moeten ze als het goed is onder elkaar staan. Dit is ook alweer gewoon semantisch beter